apm-sniffer/apm-sdk-plugin/grpc-1.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/grpc/v1/server/ServerInterceptor.java

```diff
@@ -59,15 +56,17 @@ public <REQUEST, RESPONSE> ServerCall.Listener<REQUEST> interceptCall(ServerCall
         ContextSnapshot contextSnapshot = ContextManager.capture();
         AbstractSpan asyncSpan = span.prepareForAsync();
 
-        Context context = Context.current().withValues(CONTEXT_SNAPSHOT_KEY, contextSnapshot, ACTIVE_SPAN_KEY, asyncSpan);
+        //Context context = Context.current().withValues(CONTEXT_SNAPSHOT_KEY, contextSnapshot, ACTIVE_SPAN_KEY, asyncSpan);
 
         ServerCall.Listener<REQUEST> listener = Contexts.interceptCall(
```

Comment:
As no context values added, `Context.interceptCall` may be useless.

Reply:
https://github.com/apache/skywalking-java/pull/742